core/services/chainlink/config_general.go

```diff
@@ -402,54 +401,6 @@ func (g *generalConfig) Log() config.Log {
 	return &logConfig{c: g.c.Log, rootDir: g.RootDir, level: g.logLevel, defaultLevel: g.logLevelDefault}
 }
 
-func (g *generalConfig) OCRBlockchainTimeout() time.Duration {
-	return g.c.OCR.BlockchainTimeout.Duration()
-}
-
-func (g *generalConfig) OCRContractPollInterval() time.Duration {
-	return g.c.OCR.ContractPollInterval.Duration()
-}
-
-func (g *generalConfig) OCRContractSubscribeInterval() time.Duration {
-	return g.c.OCR.ContractSubscribeInterval.Duration()
-}
-
-func (g *generalConfig) OCRKeyBundleID() (string, error) {
-	b := g.c.OCR.KeyBundleID
-	if *b == zeroSha256Hash {
-		return "", nil
-	}
-	return b.String(), nil
-}
-
-func (g *generalConfig) OCRObservationTimeout() time.Duration {
-	return g.c.OCR.ObservationTimeout.Duration()
-}
-
-func (g *generalConfig) OCRSimulateTransactions() bool {
-	return *g.c.OCR.SimulateTransactions
-}
-
-func (g *generalConfig) OCRTransmitterAddress() (ethkey.EIP55Address, error) {
-	a := *g.c.OCR.TransmitterAddress
-	if a.IsZero() {
-		return a, errors.Wrap(coreconfig.ErrEnvUnset, "OCRTransmitterAddress is not set")
-	}
-	return a, nil
-}
-
-func (g *generalConfig) OCRTraceLogging() bool {
-	return *g.c.P2P.TraceLogging
```

Comment:
@george-dorin Can we document this change in the CHANGELOG, along with instructions on what NOPs need to do to maintain the same behaviour? (i.e. set P2P.TraceLogging == OCR.TraceLogging)

Reply:
Updated Changelog